### PR TITLE
More handwritten ids

### PR DESCRIPTION
**Pull request by slevenick**
More handwritten id updates for 3.0.0 release. 
Includes:
resource_compute_region_instance_group_manager.go.erb
resource_compute_security_policy.go
resource_compute_target_pool.go
resource_container_cluster.go.erb

**Release Note Template for Downstream PRs (will be copied)**

```release-note:REPLACEME

```

### CODE REVIEW

**Comment by danawillow** [comment]
Looks like the magician got sad on your most recent commit, btw

File: third_party/terraform/resources/resource_container_cluster.go.erb [hunk 1]
```diff
@@ -2725,8 +2725,12 @@ func resourceContainerClusterStateImporter(d *schema.ResourceData, meta interfac
 		project = parts[0]
 		location = parts[1]
 		clusterName = parts[2]
+	case 6:
+		project = parts[1]
+		location = parts[3]
+		clusterName = parts[5]
 	default:
-		return nil, fmt.Errorf("Invalid container cluster specifier. Expecting {location}/{name} or {project}/{location}/{name}")
+		return nil, fmt.Errorf("Invalid container cluster specifier. Expecting {location}/{name}, {project}/{location}/{name} or projects/{project}/locations/{location}/clusters/{name}")
```
Comment (danawillow) [inline third_party/terraform/resources/resource_container_cluster.go.erb]: Can we use parseImportId here now instead of these case statements?

**Comment by slevenick** [comment]
> Looks like the magician got sad on your most recent commit, btw

My rebase caught some other commits somehow! Fixed

File: third_party/terraform/resources/resource_container_node_pool.go.erb [hunk 2]
```diff
@@ -414,30 +414,39 @@ func resourceContainerNodePoolExists(d *schema.ResourceData, meta interface{}) (
...
+		cluster = parts[2]
+		name = parts[3]
+	case 8:
+		project = parts[1]
+		location = parts[3]
+		cluster = parts[5]
+		name = parts[7]
 	default:
-		return nil, fmt.Errorf("Invalid container cluster specifier. Expecting {location}/{cluster}/{name} or {project}/{location}/{cluster}/{name}")
+		return nil, fmt.Errorf("Invalid container cluster specifier. Expecting {location}/{cluster}/{name},  {project}/{location}/{cluster}/{name} or projects/{project}/locations/{location}/clusters/{cluster}/nodePools/{name}")
```
Comment (danawillow) [inline third_party/terraform/resources/resource_container_node_pool.go.erb]: Same comment as above, also s/container cluster/container node pool/

File: third_party/terraform/tests/resource_container_cluster_test.go.erb [hunk 3]
```diff
@@ -94,13 +94,18 @@ func TestAccContainerCluster_basic(t *testing.T) {
 			},
 			{
 				ResourceName:		 "google_container_cluster.primary",
-				ImportStateIdPrefix: "us-central1-a/",
+				ImportStateId: fmt.Sprintf("us-central1-a/%s", clusterName),
```
Comment (danawillow) [inline third_party/terraform/tests/resource_container_cluster_test.go.erb]: nit: spacing

File: third_party/terraform/tests/resource_container_cluster_test.go.erb [hunk 4]
```diff
@@ -806,14 +780,39 @@ func TestAccContainerCluster_withNodeConfigScopeAlias(t *testing.T) {
...
 				ResourceName:		 "google_container_cluster.with_node_config_scope_alias",
-				ImportStateIdPrefix: "us-central1-f/",
 				ImportState:		 true,
 				ImportStateVerify:	 true,
 			},
 		},
 	})
 }
 
+<% unless version.nil? || version == 'ga' -%>
```
Comment (danawillow) [inline third_party/terraform/tests/resource_container_cluster_test.go.erb]: This got deleted in https://github.com/GoogleCloudPlatform/magic-modules/pull/2523

File: third_party/terraform/resources/resource_container_cluster.go.erb (hunk 1 above)
Comment (slevenick) [reply third_party/terraform/resources/resource_container_cluster.go.erb]: Yeah that should be easier. I tended to lean towards the case statements if we weren't setting some of the fields in state (parseImportId automatically sets state)

Setting project, region, etc shouldn't cause any issues though

Comment (slevenick) [reply third_party/terraform/resources/resource_container_cluster.go.erb]: Oh, I realized why I thought this was an issue!

Container cluster allows the user to specify either `location` or `zone`(deprecated) or `region` (deprecated)

We have removed zone and region in 3.0.0 though, so we should be good to go

File: third_party/terraform/tests/resource_container_cluster_test.go.erb [hunk 5]
```diff
@@ -93,21 +93,21 @@ func TestAccContainerCluster_basic(t *testing.T) {
 				),
 			},
 			{
-				ResourceName:		 "google_container_cluster.primary",
-				ImportStateId: fmt.Sprintf("us-central1-a/%s", clusterName),
-				ImportState:		 true,
-				ImportStateVerify:	 true,
+				ResourceName:		   "google_container_cluster.primary",
```
Comment (danawillow) [inline third_party/terraform/tests/resource_container_cluster_test.go.erb]: nit: not a big deal but i'm still seeing some spacing  weirdness here (it's a tabs/spaces thing)

**Pull request merged**